assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/style.css

```diff
@@ -0,0 +1,185 @@
+}
+
+.ee-biggie-calendar-date .ee-bcd-month {
+	font-size: calc(var(--ee-font-size-tiny) * 2);
+	font-weight: bold;
+	line-height: calc(var(--ee-font-size-tiny) * 2);
+}
+
+.ee-bcd-month-day-sep {
+	display: none;
+}
+
+.ee-biggie-calendar-date .ee-bcd-day {
+	opacity: 0.8;
+	font-size: calc(var(--ee-font-size-big) * 2);
+	font-weight: bold;
+	line-height: calc(var(--ee-font-size-big) * 2);
+	text-shadow: var(--ee-text-shadow-inset);
+}
+
+.ee-biggie-calendar-date .ee-bcd-year {
+	font-size: var(--ee-font-size-tiny);
+	letter-spacing: calc(var(--ee-letter-spacing-font-size-micro) * 2);
+	line-height: var(--ee-font-size-default);
+	left: var(--ee-padding-nano);
+	position: relative;
+}
+
+.ee-biggie-calendar-date .ee-bcd-time {
+	font-size: var(--ee-font-size-smaller);
+}
+
+.ee-biggie-calendar-date-footer {
+	font-size: var(--ee-font-size-smaller);
+	line-height: var(--ee-font-size-bigger);
+	margin: var(--ee-margin-micro) 0 0;
+	text-shadow: var(--ee-text-shadow-inset);
+	white-space: pre-line;
+}
+
+.ee-biggie-calendar-date-footer .ee-status-tag {
+	display: block;
+	font-weight: bold;
+	letter-spacing: var(--ee-letter-spacing-font-size-micro);
+	text-transform: uppercase;
+}
+
+.ee-biggie-calendar-date-bg .ee-edit-calendar-date-btn {
+	color: rgba(255, 255, 255, 0.5);
+	display: inline-block;
+}
+
+.ee-biggie-calendar-date-bg .ee-edit-calendar-date-btn:focus:enabled,
+.ee-biggie-calendar-date-bg
+	.ee-edit-calendar-date-btn:not(:disabled):not([aria-disabled='true']):not(.is-default):hover,
+.ee-biggie-calendar-date-bg
+	.ee-edit-calendar-date-btn:not(:disabled):not([aria-disabled='true']):not(.is-default):active {
+	background-color: transparent;
+	color: var(--ee-color-white);
+	box-shadow: none;
+}
+
+/* Media Queries for specific screen sizes
+ * -------------------------------------------------------------------------*/
+
+@media only screen and (max-width: 1920px) {
+}
+
+@media only screen and (max-width: 1600px) {
+}
+
+@media only screen and (max-width: 1366px) {
+}
+
+@media only screen and (max-width: 1280px) {
+}
+
+@media only screen and (max-width: 1024px) {
+}
+
+/* WordPress Admin Media Query Breakpoint: Auto-folding of the admin menu */
+@media only screen and (max-width: 960px) {
+}
+
+/* WordPress Admin Media Query Breakpoint: one column on the post write/edit screen */
+@media only screen and (max-width: 850px) {
+}
+
+/* WordPress Admin Media Query Breakpoint: Tablet */
+@media screen and (max-width: 782px) {
+}
+
+/* WordPress Admin Media Query Breakpoint: Smartphone */
+@media screen and (max-width: 600px) {
+}
+
+@media only screen and (max-width: 480px) {
+	.ee-biggie-calendar-date-bg {
+		clear: both;
+		float: none;
+		height: auto;
+		padding: var(--ee-padding-micro);
+		position: relative;
+		right: unset;
+		width: 100% !important;
+	}
+
+	.ee-biggie-calendar-date-bg.ee-bcd-pos-left {
+		float: none;
+	}
+
+	.ee-biggie-calendar-date-bg.ee-bcd-pos-right {
+		float: none;
+	}
```

Comment:
Perhaps we can use `flex-direction: row | column`? and avoid floats and clearing.

Reply:
ya OK I'll try that... I'm an older dog, but can still learn some new tricks

Reply:
I can't say I am early adopter, I was very late at this flexbox party

Reply:
and there is also the CSS grid, which I didn't completely understood: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout
***
fun fact, this :point_up: has even polyfill for IE11: http://tomrothe.de/posts/css_grid_and_ie11.html